Prevent consumers from picking up libomp.so from devel/llvm* ports

--- runtime/src/CMakeLists.txt.orig	2016-06-28 19:25:13 UTC
+++ runtime/src/CMakeLists.txt
@@ -235,6 +235,8 @@ if(WIN32)
       -o ${LIBOMP_LIB_NAME}.def ${CMAKE_CURRENT_SOURCE_DIR}/dllexports
     DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/dllexports ${LIBOMP_TOOLS_DIR}/generate-def.pl
   )
+else()
+  set_target_properties(omp PROPERTIES SOVERSION 0)
 endif()
 
 # Building the Fortran module files
